motionplan/collision.go

```diff
@@ -248,6 +256,31 @@ func (cg *collisionGraph) collisions() []Collision {
 	return collisions
 }
 
+func (cg *collisionGraph) addCollisionSpecification(specification *Collision) (err error) {
```

Comment:
[nit] This having the same name as the method down on 352 seems like it'll cause some minor confusion later. Do we have any better candidates for this method's name?

Reply:
My thought process is that in the eventual redesign CollisionSystems will be completely replaced by CollisionGraphs.  This will make it easier to use long term since you will be interacting with a specific graph rather than the whole system.  So long term I am hoping this goes away, but in the short term I still think its the right decision because CollisionSystems are just a collection of CollisonGraphs, so it would make sense that the methods for interacting with them are largely the same

Reply:
I also don't like the identically named methods.

My vote would be for this one staying the same, and the one below:
`(cs *CollisionSystem) AddCollisionSpecification`
changing to
`(cs *CollisionSystem) AddCollisionSpecificationToGraphs`